.buildkite/docker-compose.yml

```diff
@@ -2,7 +2,7 @@
 version: "3.4"
 services:
   runner-3.6: &runner-config
-    image: "python:3.6-slim"
+    image: "python:3.6"
```

Comment:
If we want to keep using slim we could add?

```apt-get update && apt-get install --no-install-recommends curl && rm -rf /var/lib/apt/lists/*```

Reply:
With caching, I don't think there's any major advantage to the slim image?

Reply:
It's mainly the docker image size difference (after pulled once on an agent it should be cached):

54.93 MB for `python:3.6-slim` https://hub.docker.com/layers/python/library/python/3.6-slim/images/sha256-f0f38f594bc1babb0a09c0323bce8fba121e278661fe124c2f2b3dc69f450022

336.9 MB for `python:3.6`https://hub.docker.com/layers/python/library/python/3.6/images/sha256-5da77e7f0723cceeeb09d82093eea3a463c63d9a873648a4e8d53643027fc8e5

Running 
```bash
$ time docker run -it python:3.6-slim sh -c 'apt-get update && apt-get install -y --no-install-recommends curl && rm -rf /var/lib/apt/lists/*'
```

Running that a few times it averages at around 7 seconds, so it's whether it's faster to pull down `281.97`MBs of extra data on the instances.

Regardless, Huon is working on adding a builder docker image image in https://github.com/stellargraph/stellargraph/pull/674 so in either case it wouldn't matter